### ISSUE TITLE
ANTICIPOS

### ISSUE THREAD

**Issue by guillermosg28** [thegreenter/greenter#234]
Hola buenas tardes,  estoy generando una boleta con anticipos igual al valor de la venta
Ejemplo:
Anticipos: 762.68
Venta: 762.68
Entonces genero la boleta con valor 0.00

![WhatsApp Image 2024-02-06 at 18 19 27](https://github.com/thegreenter/greenter/assets/6894644/b8886b26-bbd2-4050-b392-bcf8d39fde69)

Al momento de generar el resumen diario me sale este error:
            "description": "2255 - El XML no contiene el tag PaidAmount - Detalle: value='ticket: 500000105832482, error: Error en la linea: 4 :  (nodo: \"/SummaryDocuments/sac:SummaryDocumentsLine/sac:BillingPayment/cbc:PaidAmount\" valor: \"\")'",

---EN EL XML DEL RESUMEN SE ESTA ENVIANDO ASI:

<sac:SummaryDocumentsLine>
<cbc:LineID>4</cbc:LineID>
<cbc:DocumentTypeCode>03</cbc:DocumentTypeCode>
<cbc:ID>BE01-545</cbc:ID>
<cac:AccountingCustomerParty>
<cbc:CustomerAssignedAccountID>16462144</cbc:CustomerAssignedAccountID>
<cbc:AdditionalAccountID>1</cbc:AdditionalAccountID>
</cac:AccountingCustomerParty>
<cac:Status>
<cbc:ConditionCode>1</cbc:ConditionCode>
</cac:Status>
<sac:TotalAmount currencyID="PEN">0.00</sac:TotalAmount>
<cac:TaxTotal>
<cbc:TaxAmount currencyID="PEN">0.00</cbc:TaxAmount>
<cac:TaxSubtotal>
<cbc:TaxAmount currencyID="PEN">0.00</cbc:TaxAmount>
<cac:TaxCategory>
<cac:TaxScheme>
<cbc:ID>1000</cbc:ID>
<cbc:Name>IGV</cbc:Name>
<cbc:TaxTypeCode>VAT</cbc:TaxTypeCode>
</cac:TaxScheme>
</cac:TaxCategory>
</cac:TaxSubtotal>
</cac:TaxTotal>
</sac:SummaryDocumentsLine>

**Comment by Jhamnerx** [comment]
Me esta pasando igual con el taxSubTotal
parece ser como doc.mtoOperGravadas va como 0.00
en la plantilla del xml no lo esta añadiendo @giansalex  podrias actualizar para permitir eso?
Gracias
![image](https://github.com/thegreenter/greenter/assets/30221273/cccf1ab8-e797-45bb-b6ba-2421742ad6e6)

![image](https://github.com/thegreenter/greenter/assets/30221273/d273ff9a-58cb-4d0c-b929-2a4d1942331a)

mi xml no incluye el tag del taxsutotal porque ope_gravadas es 0
![image](https://github.com/thegreenter/greenter/assets/30221273/131c2601-624d-4664-bb71-b4010835a50d)

**Comment by GianmarcoCV** [comment]
si ya pagó todo que anticipo vas a deducir?

**Comment by Jhamnerx** [comment]
Esto es un tema de servicio, el cliente asi lo requiere 
por ejemplo alquiler de una camioneta se paga adelantado el mes de julio 
se emite el comrobante con anticipo y al finalizar el periodo requeire el comprobante final
son procesos que requiere el cliente y como emisor debe cumplir

**Comment by giansalex** [comment]
puede cambiarse a esta forma, para que incluya el campo cuando sea cero
```twig
{% if det.mtoOperGravadas is not null  %}
```

**Comment by Jhamnerx** [comment]
> puede cambiarse a esta forma, para que incluya el campo cuando sea cero
> 
> ```
> {% if det.mtoOperGravadas is not null  %}
> ```

Claro, eso hice modificando directo en vendor pero quiero que lo subas en una update.
Creo que no sol sería en gravadas en todo donde da 0 es considerado falso
